src/NEO/Signer.h

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../PrivateKey.h"
+#include "../Data.h"
+#include "Address.h"
+#include "Transaction.h"
+
+namespace TW::NEO {
+
+class Signer {
```

Comment:
we're unifying signer interface, can you follow this template? https://github.com/trustwallet/wallet-core/blob/master/codegen/lib/templates/newcoin/Signer.h.erb? the idea is to accept `Proto::SigningInput` so we can implement generic signer later
Thanks